scipy/spatial/transform/rotation.py

```diff
@@ -5,6 +5,7 @@
 import numpy as np
 import scipy.linalg
 from scipy._lib._util import check_random_state
+from scipy.spatial.transform._rotation_groups import create_group
```

Comment:
`from ._rotation_groups import create_group` is an alternative.

Reply:
Fixed.